gemrb/core/StringMgr.h

```diff
@@ -59,7 +59,7 @@ enum class STRING_FLAGS : uint32_t {
 };
 
 // checked against the strref itself
-#define IE_STR_ALTREF 0x0100000 // use alternate tlk file: dialogf.tlk
+#define IE_STR_ALTREF ieStrRef(0x0100000) // use alternate tlk file: dialogf.tlk
```

Comment:
Why the cast?

Reply:
as it is, it is required. one of the purposes of using `enum class` is for strictness. However, we should just define this inside the `ieStrRef` enum.

Reply:
Just looks odd for a bit, why not do it at the users?

Reply:
I don't understand your question, but I just committed a change to move it to the enum